cw/tvla.py

```diff
@@ -776,6 +776,8 @@ def run_tvla(ctx: typer.Context):
                                             x_axis)
                 for i in range(0, num_samples, sample_step_ttest))
             ttest_trace = np.concatenate((ttest_trace[:]), axis=3)
+            log.info("Saving T-test")
+            np.save('tmp/ttest.npy', ttest_trace)
```

Comment:
Oh, now I understand the problem. In CI we are doing a first call to the tvla script (which previously wouldn't update the `tmp/ttest.npy`)  including leakage detection. Since the tvla script always succeeds and we didn't want to parse  the log file to extract the pass/fail result, we added a separate leakage detection function that loads `tmp/ttest.npy`. And this was now using the old result from the previous CI test which is why you had to add this `np.save()` command here.

Reply:
An alternative solution to this would have been to call the tvla script for this particular new test with `--save_to_disk` and then adapt the CI custom leakage detection function to also operate on the `ttest-step.npy` file).

Reply:
> An alternative solution to this would have been to call the tvla script for this particular new test with `--save_to_disk` and then adapt the CI custom leakage detection function to also operate on the `ttest-step.npy` file).

Good idea. I will look into this when I implement tests for the step feature. Unfortunately, it seems that, at this moment, we can generate `ttest-step.npy` only if we are also saving leakage histograms in the output file (another strange feature of `tvla.py`). I'll add this to the tracking issue #72 to clean-up later.